orangecanvas/application/widgettoolbox.py

```diff
@@ -501,3 +503,25 @@ def acceptable(desc: Optional[WidgetDescription]) -> bool:
             FilterProxyModel.Filter(0, QtWidgetRegistry.WIDGET_DESC_ROLE,
                                     acceptable),
         ])
+        if not self.__filterText and text:
+            self.__filterText = text
+            self.__openAllTabsForFilter()
+        elif self.__filterText and not text:
+            self.__filterText = ""
+            self.__restoreAllTabsForFilter()
+
+    def __openAllTabsForFilter(self):
+        """Open all tabs for displaying filter/search results."""
+        self.__filteredSavedState = {"!__exclusive": self.exclusive()}
+        self.setExclusive(False)
+        for i in range(self.count()):
+            b = self.tabButton(i)
+            self.__filteredSavedState[b.text()] = b.isChecked()
+            b.setChecked(True)
```

Comment:
Although this line is executed, the closed tabs do not open even when they contain hits. This happens even when exclusive is `False`.

Reply:
This works for me.
![filter](https://github.com/biolab/orange-canvas-core/assets/4716745/d7a8368e-4ffd-4ea5-a944-a56097ddc107)

Reply:
Which Qt is this? I now tried on Qt6 and it works, but not on Qt5.

Reply:
Can you check again?

Reply:
This did it. Good to know.